packages/block-library/src/image/edit.native.js

```diff
@@ -236,13 +236,15 @@ class ImageEdit extends React.Component {
 						label={ __( 'Alt Text' ) }
 						value={ alt || '' }
 						valuePlaceholder={ __( 'None' ) }
-						separatorType={ 'fullWidth' }
-						onChange={ this.updateAlt }
+						separatorType={ 'none' }
+						onChangeValue={ this.updateAlt }
 					/>
+				</PanelBody>
+				<PanelBody padded={ false }>
 					<TextControl
 						label={ __( 'Clear All Settings' ) }
 						labelStyle={ styles.clearSettingsButton }
-						separatorType={ 'none' }
+						separatorType={ 'doubleFullWidth' }
```

Comment:
As far as I remember, there is some logic that would try to choose the right separator if no separatorType is specified, even though it was quite simple. And since the cell draws the separator, it wasn't trivial to account for all cases.
It would be great to improve this logic and maybe have the cell group component (PanelBody?) that can handle this.

Reply:
@tjnicolaides Logic for choosing the proper separator [exists](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/bottom-sheet/cell.native.js#L81-#L96) in `BottomSheetCell`. 

`PanelBody` is just a container for each section, where within are `TextControls` based on mentioned above cell.

Reply:
> Logic for choosing the proper separator exists in BottomSheetCell.

That's what I mean, since it's in the cell, it's difficult to know if the cell is at the end to not draw the separator (for example). So we could add that logic to the container, who has that information.

This is just an idea to apply in the future 👍

Reply:
>  I don't think the block code should have to care which separator will be used. But I think we can work on that issue separately.

Agreed. It looks like we should revisit the API here. It is also good for making this cross-platform, I don't see separatorType prop on web. 🤔 @etoledom could you be able to open a tech debt issue for this?

Reply:
https://github.com/wordpress-mobile/gutenberg-mobile/issues/1421